etc/docker/docker_entrypoint.sh

```diff
@@ -89,8 +89,8 @@ NAT64_PREFIX=${NAT64_PREFIX/\//\\\/}
 sed -i "s/^prefix.*$/prefix $NAT64_PREFIX/" /etc/tayga.conf
 sed -i "s/dns64.*$/dns64 $NAT64_PREFIX {};/" /etc/bind/named.conf.options
 
-echo "OTBR_AGENT_OPTS=\"-I $TUN_INTERFACE_NAME -B $BACKBONE_INTERFACE -d7 $RADIO_URL\"" >/etc/default/otbr-agent
-echo "OTBR_WEB_OPTS=\"-I $TUN_INTERFACE_NAME -d7 -p 80\"" >/etc/default/otbr-web
+echo "OTBR_AGENT_OPTS=\"-I $TUN_INTERFACE_NAME -B $BACKBONE_INTERFACE -d4 $RADIO_URL\"" >/etc/default/otbr-agent
+echo "OTBR_WEB_OPTS=\"-I $TUN_INTERFACE_NAME -d4 -p 80\"" >/etc/default/otbr-web
```

Comment:
why do we lower the log level?

Reply:
Current otbr-agent directly uses the syslog levels as its log levels, it supports the level `0~7`. This CL redefined the otbr-agent log levels just like the OpenThread log levels, the otbr-agent log level becomes `0~4`. So the value of the log level is changed here.